File: common/src/main/kotlin/org/bitlap/common/exception/BSQLException.kt
package org.bitlap.common.exception

/**
 * driver exception
 * please use this when we do not need error stack.
 * @author 梦境迷离
 * @since 2021/6/6
 * @version 1.0
 */
data class BSQLException(val msg: String = "SQL Exception") : BitlapException(msg)

File: network/src/main/scala/org/bitlap/net/BSQLException.scala
package org.bitlap.net

import java.sql.SQLException

/**
 *
 * @author 梦境迷离
 * @since 2021/11/20
 * @version 1.0
 */
case class BSQLException(msg: String = "Bitlap SQL Exception", cause: Throwable = null) extends SQLException(msg, cause)
